### ISSUE TITLE
Fix warnings in Julia v0.7

### ISSUE THREAD

**Issue by juliohm** [JuliaStats/Rmath.jl#51]
How to fix these? I tried to just add `@` in front of `cfunction`, but it causes another error:

```julia
┌ Warning: The function `cfunction` is now written as a macro `@cfunction`.
│   caller = __init__() at Rmath.jl:47
└ @ Rmath Rmath.jl:47
┌ Warning: The function `cfunction` is now written as a macro `@cfunction`.
│   caller = __init__() at Rmath.jl:49
└ @ Rmath Rmath.jl:49
┌ Warning: The function `cfunction` is now written as a macro `@cfunction`.
│   caller = __init__() at Rmath.jl:51
└ @ Rmath Rmath.jl:51
```

**Comment by ararslan** [comment]
There is no Compat support for the 0.7-style `@cfunction` yet.

**Comment by dmbates** [comment]
Closed by #52
